src/openforms/js/components/admin/form_design/logic/actions/Actions.js

```diff
@@ -347,4 +348,37 @@ ActionComponent.propTypes =
       onChange: PropTypes.func.isRequired,
     };
 
+export const detectProblems = (action, intl) => {
+  switch (action.action.type) {
```

Comment:
I would use a simple if statement here since we have only one case, do we use switch for possible update in the code?

Reply:
yeah, I'm doing it for consistency and future problem detection in other action types!